palisade-service/src/test/java/uk/gov/gchq/palisade/service/palisade/service/ResourceServiceTest.java

```diff
@@ -101,7 +101,7 @@ public void infoOnGetResourcesRequest() {
         GetResourcesByIdRequest request = new GetResourcesByIdRequest().resourceId("/path/to/bob_file.txt");
         request.setOriginalRequestId(new RequestId().id("Original ID"));
         Map<LeafResource, ConnectionDetail> response = Mockito.mock(Map.class);
-        Mockito.when(resourceClient.getResourcesById(Mockito.any(), Mockito.eq(request))).thenReturn(response);
+//        Mockito.when(resourceClient.getResourcesById(Mockito.any(), Mockito.eq(request))).thenReturn(response);
```

Comment:
assuming this was commented out because the test fails here, can you please fix the test

Reply:
This was commented out because SQ reported that it was unnecessary code - i.e it is not used by the test, at all. I will delete the line.